operatorhub/openshift/release-artifacts/bundle/manifests/openshift-pipelines-operator-rh.clusterserviceversion.yaml

```diff
@@ -978,7 +978,7 @@ spec:
     name: IMAGE_ADDONS_PARAM_BUILDER_IMAGE
   - image: registry.redhat.io/openshift-serverless-1/client-kn-rhel8@sha256:37109f6de75dda750496f6668164d0dceaf9db5278e8c034708436f39de3276e
     name: IMAGE_ADDONS_PARAM_KN_IMAGE
-  - image: registry.redhat.io/rhel8/skopeo@sha256:7fd65200b4847fc84fba0466519b87f915e28c0e9bc90d3022beecfebffd4af8
+  - image: registry.redhat.io/rhel9/skopeo@sha256:993f799d58c48151ecae7a84e63860f6c0039e8e18efed5ad67e7752f4b7e408
     name: IMAGE_ADDONS_SKOPEO_COPY
```

Comment:
After this modification, it is exactly the same as the content on line 1018; shouldn't we keep just one instance?
https://github.com/tektoncd/operator/blob/3e681836aafb97082709cf34c8d8e7d5d1562b9c/operatorhub/openshift/release-artifacts/bundle/manifests/openshift-pipelines-operator-rh.clusterserviceversion.yaml#L1017-L1018

Reply:
Ya :+1: will remove duplicate entry from related images section